js/src/main/scala/crystal/react/implicits/package.scala

```diff
@@ -250,47 +250,71 @@ package object implicits {
   }
 
   implicit class PotRender[A](val pot: Pot[A]) extends AnyVal {
-    def renderPending(f: Long => VdomNode): VdomNode =
+    def renderPending(f: => VdomNode): VdomNode =
```

Comment:
This seems good, I never got what the time was useful for

Reply:
It comes from the original `Pot` in `diode`. The idea is that you can render different things if the pending operation is taking too long, (eg: `Still working on it..` if it takes more than 5 seconds). But we are not using this functionality.